documentation/partials/api/_slots.pug

```diff
@@ -30,6 +30,12 @@ h2.typo__h2#sub-slots(data-section) Slots
             | Shows when no elements match the search query. Defaults to string:
             br
             strong  Default: No elements found. Consider changing the search query.
+        tr.table__trq
```

Comment:
Should this be `tr.table__tr` ? I tried searching the docs for this specific option, couldn't find it - so maybe this is the reason?

Reply:
#872